personalities/sctfe/config.go

```diff
@@ -140,20 +140,28 @@ func ValidateLogConfig(cfg *configpb.LogConfig, origin string, projectID string,
 		return nil, errors.New("empty spannerDB")
 	}
 
+	lExtKeyUsages := []string{}
+	lRejectExtensions := []string{}
+	if extKeyUsages != "" {
+		lExtKeyUsages = strings.Split(extKeyUsages, ",")
+	}
+	if rejectExtensions != "" {
+		lRejectExtensions = strings.Split(rejectExtensions, ",")
+	}
+
```

Comment:
I think this is equivalent to:

```suggestion
		lExtKeyUsages := strings.Split(extKeyUsages, ",")
		lRejectExtensions := strings.Split(rejectExtensions, ",")
```

In which case they could then also be inlined into the instantiation of the `ValidatedLogConfig` struct below.

Reply:
I thought as well, but unfortunately that's not the case: see https://go.dev/play/p/aN9KQ0ljiu6 and https://pkg.go.dev/strings#Split.

Reply:
:(